### PR TITLE
Fix data-disablescrolling=false not working

### PR DESCRIPTION
**Pull request by nicofrand**

### CODE REVIEW

File: js/offcanvas.js [hunk 1]
```diff
@@ -36,14 +36,14 @@
     if (this.options.autohide && !this.options.modal) {
       var eventName = (navigator.userAgent.match(/(iPad|iPhone)/i) === null) ? 'click' : 'touchstart'
       $(document).on('click touchstart', $.proxy(this.autohide, this))
-    }
-
-    if (this.options.toggle) this.toggle()
+    }   
 
-    if (this.options.disablescrolling) {
+    if (typeof(this.options.disablescrolling) === "boolean") {
```
Comment (jasny) [inline js/offcanvas.js]: If you use `data-disablescrolling="false"`, the option will be a string and not a boolean. This doesn't seem to fix the issue.

This also breaks setting `"disablescrolling": false` in JavaScript. `false` is a boolean, so scrolling will be disabled, but that's as intended.

**Comment by nicofrand** [comment]
You're using jQuery .Data() which converts "false" to false (boolean).


I tested it before making the PR.


I don't understand the part about setting it in JS ? Currently you could only set it to true even though this was the default value..

File: js/offcanvas.js (hunk 1 above)
Comment (nicofrand) [inline js/offcanvas.js]: You're using jQuery .data() which converts "false" to false (boolean) automatically.

And if you pass "disablescrolling": false in JS, everything works fine, I am not sure what you meant here ?

I tested this PR and this works fine for me.

**Comment by jasny** [comment]
@nicofrand This change is no good. It disables scrolling regardless of setting `disablescrolling` to `true` or `false`.

console bs 503

**Comment by nicofrand** [comment]
What ? But I only test that it is a boolean, the value that is affected is here : https://github.com/nicofrand/bootstrap/blob/a4f103dddeb2b7f6101479328fa47910b9f4316d/js/offcanvas.js#L42.

If you pass `disablescrolling` as `false`, then it checks it is a boolean and then affects the value (either `true` or `false`).

**Comment by jasny** [comment]
yes, true... I should have taken a closer look at the code. Sorry about that.

**Comment by nicofrand** [comment]
Sorry to bother again but there was an issue during the merge, the following line should be removed: https://github.com/jasny/bootstrap/blob/master/js/offcanvas.js#L48. (since it is now at the end as needed).

**Pull request merged**